src/commands/general/guildInfo.ts

```diff
@@ -0,0 +1,19 @@
+import Command, { SendFunction } from '../../util/BaseCommand';
+import SentinelClient from '../../client/SentinelClient';
+import { Message } from 'discord.js';
+import CommandArguments from '../../util/CommandArguments';
+
+export default class guildStatsCommand extends Command {
```

Comment:
```suggestion
export default class GuildStatsCommand extends Command {
```
class names should start with a capital letter